cogs/send_get_roles_reminders.py

```diff
@@ -156,6 +156,13 @@ async def send_get_roles_reminders(self) -> None:
             if time_since_role_received <= datetime.timedelta(days=1):
                 continue
 
+            if member not in guild.members:
+                    logger.info(
+                        "Member: %s left the server before the introduction "
+                        "reminder could be sent.",
+                    ), member
+                    repr(member.id)
+
```

Comment:
A `continue` statement is required so that the function does not continue with sending the message even when this check found that the message should not be sent